components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -1457,6 +1462,29 @@ private UsersGetResponse filterUsersBySingleAttribute(ExpressionNode node, Map<S
         return getDetailedUsers(filteredUsers, totalResults);
     }
 
+    /**
+     * method to get user count by filtering parameter.
+     *
+     * @param node       Expression node for single attribute filtering
+     * @param offset     Starting index of the count
+     * @param limit      Counting value
+     * @param sortBy     SortBy
+     * @param sortOrder  Sorting order
+     * @param domainName Domain to run the filter
+     * @return User count
+     * @throws BadRequestException
```

Comment:
Describe exceptions in all places

Reply:
Resolved with https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/547/commits/6971d7032dd66875fb60641349cbae8585474a28